Tasks/DownloadFileShareArtifactsV0/main.ts

```diff
@@ -0,0 +1,105 @@
+        taskDisplayName: tl.getVariable('TASK.DISPLAYNAME'),
+        jobid: tl.getVariable('SYSTEM.JOBID'),
+        agentVersion: tl.getVariable('AGENT.VERSION'),
+        agentOS: tl.getVariable('AGENT.OS'),
+        agentName: tl.getVariable('AGENT.NAME'),
+        version: taskJson.version
+    };
+}
+
+function publishEvent(feature, properties: any): void {
+    try {
+        var splitVersion = (process.env.AGENT_VERSION || '').split('.');
+        var major = parseInt(splitVersion[0] || '0');
+        var minor = parseInt(splitVersion[1] || '0');
+        let telemetry = '';
+        if (major > 2 || (major == 2 && minor >= 120)) {
+            telemetry = `##vso[telemetry.publish area=${area};feature=${feature}]${JSON.stringify(Object.assign(getDefaultProps(), properties))}`;
+        }
+        else {
+            if (feature === 'reliability') {
+                let reliabilityData = properties;
+                telemetry = "##vso[task.logissue type=error;code=" + reliabilityData.issueType + ";agentVersion=" + tl.getVariable('Agent.Version') + ";taskId=" + area + "-" + JSON.stringify(taskJson.version) + ";]" + reliabilityData.errorMessage
+            }
+        }
+        console.log(telemetry);;
+    }
+    catch (err) {
+        tl.warning("Failed to log telemetry, error: " + err);
+    }
+}
+
+async function main(): Promise<void> {
+    var promise = new Promise<void>(async (resolve, reject) => {
+        var downloadPath: string = tl.getInput("downloadPath", true);
+        var debugMode: string = tl.getVariable('System.Debug');
+        var isVerbose: boolean = debugMode ? debugMode.toLowerCase() != 'false' : false;
+        var parallelLimit: number = +tl.getInput("parallelizationLimit", false);
+        var retryLimit = parseInt(tl.getVariable("VSTS_HTTP_RETRY")) ? parseInt(tl.getVariable("VSTS_HTTP_RETRY")) : 4;
+        var itemPattern: string = tl.getInput("itemPattern", false) || '**';
+
+        let downloader = new engine.ArtifactEngine();
+        let downloadUrl = tl.getInput("location", true);
+        let artifactName = tl.getInput("artifactName", true);
+        let artifactLocation = path.join(downloadUrl, artifactName);
+
+        console.log(tl.loc("DownloadArtifacts", artifactName, artifactLocation));
+        if (!fs.existsSync(artifactLocation)) {
+            console.log(tl.loc("ArtifactNameDirectoryNotFound", artifactLocation, downloadUrl));
+            artifactLocation = downloadUrl;
+        }
+
+        let downloaderOptions = new engine.ArtifactEngineOptions();
+        downloaderOptions.itemPattern = itemPattern;
+        downloaderOptions.verbose = isVerbose;
+
+        if (parallelLimit) {
+            downloaderOptions.parallelProcessingLimit = parallelLimit;
+        }
+
```

Comment:
add this console log?
https://github.com/Microsoft/vsts-tasks/blob/fb86546adc1dfeca3602a2887891f893dc732823/Tasks/DownloadBuildArtifactsV0/main.ts#L258

Reply:
it is already logged above.